automation/test.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+# CI considerations: $TARGET is used by the jenkins build, to distinguish what to test
+# Currently considered $TARGET values:
+#     kubernetes-release: Runs all functional tests on a release kubernetes setup
+#     openshift-release: Runs all functional tests on a release openshift setup
+
+set -ex
+
+export WORKSPACE="${WORKSPACE:-$PWD}"
+readonly ARTIFACTS_PATH="$WORKSPACE/exported-artifacts"
+
+if [[ $TARGET =~ openshift-.* ]]; then
+  export KUBEVIRT_PROVIDER="os-3.11.0-multus"
+else
+  export KUBEVIRT_PROVIDER="k8s-multus-1.12.2"
+fi
+
+export KUBEVIRT_NUM_NODES=2
+
+wait_for_download_lock() {
+  local max_lock_attempts=60
+  local lock_wait_interval=60
+
+  for ((i = 0; i < $max_lock_attempts; i++)); do
+      if (set -o noclobber; > $1) 2> /dev/null; then
+          echo "Acquired lock: $1"
+          return
+      fi
+      sleep $lock_wait_interval
+  done
+  echo "Timed out waiting for lock: $1" >&2
+  exit 1
+}
+
+release_download_lock() {
+  if [[ -e "$1" ]]; then
+    rm -f "$1"
+    echo "Released lock: $1"
+  fi
+}
+kubectl() { cluster/kubectl.sh "$@"; }
+
+export NAMESPACE="${NAMESPACE:-kube-system}"
+
+# Make sure that the VM is properly shut down on exit
+trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
+
+make cluster-down
+make cluster-up
+
+# Wait for nodes to become ready
+set +e
+kubectl get nodes --no-headers
+kubectl_rc=$?
+while [ $kubectl_rc -ne 0 ] || [ -n "$(kubectl get nodes --no-headers | grep NotReady)" ]; do
+    echo "Waiting for all nodes to become ready ..."
+    kubectl get nodes --no-headers
+    kubectl_rc=$?
+    sleep 10
+done
+set -e
+
+echo "Nodes are ready:"
+kubectl get nodes
+
+# OpenShift is running important containers under default namespace
+namespaces=(kube-system default)
+if [[ $NAMESPACE != "kube-system" ]]; then
+  namespaces+=($NAMESPACE)
+fi
+
+# Run cluster-sync to deploy bridge-marker on the nodes
+make cluster-sync
+
+timeout=300
+sample=30
+
+for i in ${namespaces[@]}; do
+  # Wait until kubevirt pods are running
```

Comment:
we don't deploy any kubevirt pods, this block is not needed

Reply:
Removed